.github/workflows/integration.yml

```diff
@@ -218,7 +254,9 @@ jobs:
         run: |
           cd Pilot
           export VO_LHCB_SW_DIR=${GITHUB_WORKSPACE}/Pilot
-          curl https://lhcbdirac.s3.cern.ch/Pilot3/LHCbPilotCommands.py -o LHCbPilotCommands.py
+          
+          curl https://gitlab.cern.ch/fstagni/LHCbPilot/-/raw/debug/LHCbPilot/LHCbPilotCommands.py -o LHCbPilotCommands.py
```

Comment:
Note to myself : fix this once https://gitlab.cern.ch/lhcb-dirac/LHCbPilot/-/merge_requests/52 is merged